### ISSUE TITLE
Evidence strings from some data sources contain duplication of literature

### ISSUE THREAD

**Issue by peatroot** [opentargets/platform#290]
Looking at the **ot-releases/18.10** data files, there are several data sources which appear to duplicate their literature references, ie. `.literature` and `.evidence.provenance_type.literature` are identical.

The counts below were calculated with `jq` using, for example:
```
gzcat 18.10_eva-01-10-2018.json.gz | sed '1s/^/[/; $!s/$/,/; $s/$/]/' | jq -c 'map(select(.literature)) | length'
gzcat 18.10_eva-01-10-2018.json.gz | sed '1s/^/[/; $!s/$/,/; $s/$/]/' | jq -c 'map(select(.evidence.provenance_type.literature)) | length'
```

| Data Source     | Total JSON strings | Number with .literature | Number with .evidence.provenance_type.\*.literature           |
| --------------- | ------------------ | ----------------------- | ------------------------------------------------------------- |
| ChEMBL          | 465039             | 252375                  | 252375 (.evidence.target2drug.provenance_type.literature)     |
| ChEMBL (19.04)  |    388530   |      0      | 192732 (.evidence.target2drug.provenance_type.literature)     |
| UniProt (19.04)    | 33667        | 4806    | 4806 (27783 evidence strings have gene2variant & variant2disease specific literature)  |
| EVA             | 85106              | 70225                   | 2775                                                          |
| Reactome        | 10076              | 10018                   | 10018                                                         |
| PheWAS Catalog  | 90038              | 0                       | 0                                                             |
| Atlas (19.04)      | 385954             | 346023                     | 346023                                                           |
| Cosmic          | 58001              | 43721                   | 43721                                                         |
| GWAS            | 126789             | 126789                  | 126789 (.evidence.variant2disease.provenance_type.literature) |
| SlapEnrich      | 90431              | 0                       | 90431                                                         |
| EuropePMC       | 6303570            | yes                     | yes                                                           |
| GenomicsEngland | 11079              | 0                       | 11079                                                         |
| Intogen         | 2594               | 0                       | 2594                                                          |
| Phenodigm       | 506293             | 0                       | 0                                                             |
| Progeny         | 407                | 0                       | 405                                                           |

**Comment by MichaelaEBI** [comment]
I started updating the table above but realised that things are more complex for some data sources. I have  started a [Google Sheet](https://docs.google.com/spreadsheets/d/1h3KbT_1Fa1DeuIn7YX2Vf-XNfQdcDjULk32f2eT4LCA/edit#gid=0) that I will use to document where in the JSON evidence strings literature references are.

With @gkos-bio and @LucaFumis we established that `'literature` does not seem to be used by the front end. ChEMBL has already removed the `.literature` section from their evidence strings and everything seems to work fine. When I finish documenting which data source has literature where in the evidence strings, I will check again that `.literature` is not used in the front end and I will then approach the different data sources to ask them to remove `.literature` for 19.06.
